### ISSUE TITLE
Suggestion: Display error when URL is invalid

### ISSUE THREAD

**Issue by muttleyxd** [mrin9/RapiDoc#41]
If specified URL returns 404, then RapiDoc will display 'Loading ...' message forever.

I'm not an expert on JS, but I think it should be possible to display an error message saying 'URL % returned 404'

HTML code to trigger it:
```
  <!doctype html> <!-- Important: must specify -->
  <html>
  <head>
    <meta charset="utf-8"> <!-- Important: rapi-doc uses utf8 charecters -->
    <script src="https://unpkg.com/rapidoc/dist/rapidoc-min.js"></script>
  </head>
  <body>
    <rapi-doc 
      spec-url="https://google.com/give_me_yaml_please.yaml" 
    > </rapi-doc>
  </body> 
  </html>
```

![Screenshot from 2019-07-03 08-59-25](https://user-images.githubusercontent.com/9843181/60570000-e8414380-9d70-11e9-8a06-1816000c1fa7.png)

**Comment by mrin9** [comment]
we can definitely improve the experience around this area. Will come up with something to address it

**Comment by mrin9** [comment]
fixed in 2d0dcff192c9a896070947ff0c0c04bcf0cde7ad
